src/features/tutorials/components/tutorial-form/index.tsx

```diff
@@ -0,0 +1,181 @@
+
+interface TutorialFromProps {
+  defaultValues?: TutorialPayload;
+  onSubmit: (data: TutorialPayload) => void;
+  isSubmitting: boolean;
+}
+
+export function TutorialForm({
+  defaultValues,
+  onSubmit,
+  isSubmitting,
+}: TutorialFromProps) {
+  const isEditing = useMemo(() => Boolean(defaultValues), [defaultValues]);
+  let nomeArquivo;
+  const {
+    register,
+    handleSubmit,
+    control,
+    formState: { errors },
+  } = useForm({
+    defaultValues: {
+      ...defaultValues,
+    },
+  });
+
+  const { data: categories, isLoading: isLoadingCategories } =
+    useGetAllCategoryTutorial();
+
+  const categoryOptions = categories?.map((category) => ({
+    label: category?.name,
+    value: category?.id,
+  }));
+
+  const [fileName, setFileName] = useState<File>();
+  const inputRef = useRef();
+
+  const handleDragOver = (event: any) => {
+    event.preventDefault();
+  };
+  const handleDrop = (event: any) => {
+    event.preventDefault();
+    // console.log(event.dataTransfer.files[0]);
+    setFileName(event.dataTransfer.files[0]);
+  };
+
+  if (fileName) {
+    nomeArquivo = fileName.name;
+  }
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)} encType="multipart/form-data">
+      <Input
+        label="Nome do tutorial"
+        {...register('name', { required: 'Campo obrigatório' })}
+        placeholder="Digite o nome do tutorial"
+        errors={errors?.name}
+      />
+
+      {/* <Input
+        label="ID da categoria"
+        {...register('category_id', { required: 'Campo obrigatório' })}
+        placeholder="Digite o id da categoria"
+        errors={errors?.name}
+      /> */}
+
+      <ControlledSelect
+        control={control}
+        name="category_id"
+        id="category_id"
+        options={categoryOptions}
+        isLoading={isLoadingCategories}
+        placeholder="Categoria"
+        label="Categoria"
+      />
+
+      <div
+        style={{
+          paddingTop: 20,
+          paddingBottom: 20,
+        }}
+        onDragOver={handleDragOver}
+        onDrop={handleDrop}
+      >
+        <span
+          aria-controls="input"
+          className="file"
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            paddingTop: 50,
+            paddingBottom: 50,
+            alignItems: 'center',
+            justifyContent: 'center',
+            border: '1px dotted white',
+            borderRadius: 6,
+          }}
+        >
+          <GrDocumentPdf
+            size="40"
+            style={{
+              alignContent: 'center',
+              filter: 'invert(0.6)',
+            }}
+          />
+          <Input
+            id="fileinput"
+            label=""
+            type="file"
+            // Call handleFile function when a file is selected before uploading
+            {...register('file', { required: 'Campo obrigatório' })}
+            placeholder="Escolha um arquivo e jogue"
+            errors={errors?.name}
```

Comment:
O correto é **file** ao invés de **name**
```suggestion
            errors={errors?.file}
```